node_modules/oae-core/upload/js/upload.js

```diff
@@ -399,7 +399,6 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
                 'url': '/api/content/create',
                 'dropZone': $('#upload-dropzone', $rootel),
                 'forceIframeTransport': useIframeTransport,
-                'replaceFileInput': false,
```

Comment:
It's not a problem on uploadnewversion because that widget doesn't present an intermediate dialog for confirmation. It simply uploads the new file immediately. It's also not a problem with importusers. I'm not sure exactly why, but the form control layout is significantly different and that's apparently enough to prevent triggering the Firefox bug.

We could make the changes in those widgets as well for consistency, but from a functional viewpoint they aren't required.